libs/sdk-ui/src/base/results/facade.ts

```diff
@@ -9,10 +9,10 @@ import { IDataAccessMethods } from "./dataAccess";
 import { newDataAccessMethods } from "./internal/dataAccessMethods";
 
 /**
- * This wrapper for {@link IDataView} provides various convenience methods to work with data and metadata stored inside
- * the provided instance of {@link IDataView}.
+ * This wrapper for `IDataView` provides various convenience methods to work with data and metadata stored inside
```

Comment:
Why remove the links? If this was unresolvable (because it is in another package), you can use the hash syntax, for example:

```
{@link @gooddata/sdk-backend-spi#IDataView}
```

This has the added benefit that it checks that the `@gooddata/sdk-backend-spi` has a member called `IDataView` exported (hence some basic interpackage check).

Reply:
There are examples on how to write the links in the docs: https://tsdoc.org/pages/tags/link/

Reply:
i did not know that.. let me fix that

Reply:
ok fixed together with other files where i removed the links